firmware/boron-ins-fsm/src/stateMachine.h

```diff
@@ -22,6 +22,7 @@
 //length of time between debug publishes
 #define DEBUG_PUBLISH_INTERVAL 1500  //ms
 #define SM_HEARTBEAT_INTERVAL 600000  //ms = 10 min
+#define NINE_MINS 540000  //ms = 9 min
```

Comment:
In general, it is bad practice name variables or constants based on their value. 

If, in a few weeks, we decide that we want this to wait 8 minutes instead of 9, you would have the choice to either change the value and not the name, and thus be confusing to anyone reading the code later. Or you need to change both the value and the name, and lose most of the benefits of having made it a constant in the first place.

What is a more descriptive name you could use instead of `NINE_MINS`?

Reply:
Sounds good, I am going to name it to DEVICE_RESET_THRESHOLD.